src/event.rs

```diff
@@ -156,7 +178,7 @@ impl<N: Notify> Processor<N> {
     /// Handle events from glutin
     ///
     /// Doesn't take self mutably due to borrow checking. Kinda uggo but w/e.
-    fn handle_event<'a>(
+    pub fn handle_event<'a>(
```

Comment:
I don't think this should be `pub` just for tests. Tests for this system can just be added in the same file.